src/test/java/hw2/ex2/SecondExerciseTest.java

```diff
@@ -0,0 +1,166 @@
+        // and check that drop down contains options
+        WebElement serviceHeaderSubMenu = driver.findElement(By.
+                xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']/li[@class='dropdown']"));
+        serviceHeaderSubMenu.click();
+        WebElement serviceSupportSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Support')]"));
+        softAssert.assertTrue(serviceSupportSubCategory.isDisplayed());
+        WebElement serviceDatesSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Dates')]"));
+        WebElement serviceComplTableSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Complex Table')]"));
+        WebElement serviceSimplTableSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Simple Table')]"));
+        WebElement serviceTableWithPagesSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Table with pages')]"));
+        WebElement serviceDiffElemSubCategory =
+                driver.findElement(By.xpath("//ul[@class='dropdown-menu']//li//a[contains(.,'Different elements')]"));
+        softAssert.assertTrue(serviceDatesSubCategory.isDisplayed());
+        softAssert.assertTrue(serviceComplTableSubCategory.isDisplayed());
+        softAssert.assertTrue(serviceSimplTableSubCategory.isDisplayed());
+        softAssert.assertTrue(serviceTableWithPagesSubCategory.isDisplayed());
+        softAssert.assertTrue(serviceDiffElemSubCategory.isDisplayed());
+        softAssert.assertAll();
+
+        // 6. Click on Service subcategory in the left section
+        // and check that drop down contains options
+        WebElement serviceHeaderLeftPanel = driver.findElement(By.
+                xpath("//li[@class='menu-title']//a[contains(.,'Service')]"));
+        serviceHeaderLeftPanel.click();
+        WebElement serviceSupportSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Support')]"));
+        WebElement serviceDatesSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Dates')]"));
+        WebElement serviceComplTableSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Complex Table')]"));
+        WebElement serviceSimplTableSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Simple Table')]"));
+        WebElement serviceTableWithPagesSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Table with pages')]"));
+        WebElement serviceDiffElemSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Different elements')]"));
```

Comment:
Is it possible collapse these elements to the loop and use `List<WebElement>`

Reply:
fixed